### PR TITLE
Added buildings improvments in the doc

### PR DESCRIPTION
**Pull request by Francois2**
Linked to omichel/webots#5542

### CODE REVIEW

**Comment by Francois2** [comment]
:thumbsup:

File: automobile/buildings.md [hunk 1]
```diff
@@ -77,7 +79,7 @@ roof).
 ### SimpleBuilding PROTO
 
 The `SimpleBuilding` PROTO is similar to the `Building` one except that the wall and roof appearances don't need to be set.
-The `wallAppearance` and `roofAppearance` fields are replaced by the `wallType` and `roofType` ones.
+The `wallAppearance` and `roofAppearance` fields are replaced by the `wallType` and `roofType` ones. Some wall types and roof types supported colors. So there is two other parameters : `wallColor` and `roofColor`.
```
Comment (DavidMansolino) [inline automobile/buildings.md]: types supported colors => supports

I would rephrase it something like this:
The `wallColor` and `roofColor` fields define respectively the color of the wall and roof. Note that not all the wall and roof types support the color argument.

File: automobile/buildings.md [hunk 2]
```diff
@@ -88,6 +90,10 @@ The available roof types are:
   - metal tiles
   - bitumen
 
+The available colored roof types are:
+  - tiled
+  - bitumen
```
Comment (DavidMansolino) [inline automobile/buildings.md]: Instead of duplicating the list I would put in parenthesis in the first list `(supports color)`.

**Comment by Francois2** [comment]
👍

File: automobile/buildings.md [hunk 3]
```diff
@@ -63,11 +64,12 @@ Building {
 - `name`: Could contain the building name.
 - `floorHeight`: Defines the height of one floor.
 - `floorNumber`: Defines the number of floors (excluding roof).
+- `startingFloor`: Defines the number of floors between the ground and the first floor.
```
Comment (DavidMansolino) [inline automobile/buildings.md]: I would say: Defines the number of floors -> Defines the number of empty levels

File: automobile/buildings.md [hunk 4]
```diff
@@ -77,16 +79,16 @@ roof).
 ### SimpleBuilding PROTO
 
 The `SimpleBuilding` PROTO is similar to the `Building` one except that the wall and roof appearances don't need to be set.
-The `wallAppearance` and `roofAppearance` fields are replaced by the `wallType` and `roofType` ones.
+The `wallAppearance` and `roofAppearance` fields are replaced by the `wallType` and `roofType` ones. The `wallColor` and `roofColor` fields define respectively the color of the wall and roof. Note that not all the wall and roof types support the color argument.
```
Comment (tn12787) [inline automobile/buildings.md]: ...fields define respectively the color of the wall and roof -> fields define the color of the wall and roof, respectively

I'd also change "Note that" to "Bear in mind that", as we tend to (over)use "Note that" elsewhere in the documentation.

File: automobile/buildings.md (hunk 3 above)
Comment (tn12787) [inline automobile/buildings.md]: I would say "Defines the floor number for the "ground floor" of the building, as not all buildings start at the ground floor.".

**Comment by Francois2** [comment]
👍
